core/templates/pages/exploration-editor-page/feedback-tab/services/thread-data-backend-api.service.ts

```diff
@@ -165,11 +171,12 @@ export class ThreadDataBackendApiService {
         let feedbackThreadBackendDicts = threadData.feedback_thread_dicts;
         let suggestionThreadBackendDicts = threadData.suggestion_thread_dicts;
 
-        let suggestionBackendDictsByThreadId = new Map(
-          suggestionBackendDicts.map(dict => [
-            this.suggestionsService.getThreadIdFromSuggestionBackendDict(dict),
-            dict
-          ]));
+        let suggestionBackendDictsByThreadId:
+         Map<string | null, SuggestionBackendDict> = new Map(
+           suggestionBackendDicts.map(dict => [
+             this.suggestionsService.getThreadIdFromSuggestionBackendDict(dict),
+             dict
+           ]));
```

Comment:
Also should we create type variable?

Reply:
Not sure about this comment, can you please rephrase or give an example ?

Reply:
Sorry, create type variable for the `Map<string | null, SuggestionBackendDict>`.

Reply:
done